src/linker/Linker.Steps/BlacklistStep.cs

```diff
@@ -91,10 +91,10 @@ protected override void Process ()
 									.Where (res => res.Name.Equals ("ILLink.LinkAttributes.xml", StringComparison.OrdinalIgnoreCase))
 									.Cast<EmbeddedResource> ()) {
 					try {
-						Context.LogMessage (MessageContainer.CreateInfoMessage ($"Processing embedded {rsc.Name} from {asm.Name}"));
+						Context.LogMessage ($"Processing embedded {rsc.Name} from {asm.Name}");
 						steps_to_add.Push (GetExternalLinkAttributesStep (rsc, asm));
 					} catch (XmlException ex) {
-						Context.LogMessage (MessageContainer.CreateErrorMessage ($"Error processing {rsc.Name} from {asm.Name}: {ex}", 1003));
+						Context.LogError ($"Error processing {rsc.Name} from {asm.Name}: {ex}", 1003);
```

Comment:
Since we've segmented the diagnostics into default severity by error code, shouldn't we know what the default severity should be just by looking at the code? Why do we have to say what it is at the callsite (i.e. LogError vs LogMessage)

Reply:
Good point. We could replace every `LogWarning`/`LogError` by a single `Log` and then just check the code... at the same time, I also think that it doesn't hurt to be explicit about what's the thing that we're logging (error or warning.)

Reply:
Opened #1406 to track this.